### DISCUSSION TITLE
smallbaselineAPP_aria.ipynb in the tutorial of EarthScope2023，the SanFranSenDT42.tar downloaded  from zenodo by 'wget' can't be unzip

### DISCUSSION THREAD

**Discussion by abcdedfde** [insarlab/MintPy#1184] [Q&A]
**Description of the problem**

smallbaselineAPP_aria.ipynb in the tutorial of EarthScope2023，the SanFranSenDT42.tar downloaded  from zenodo by 'wget' can't be unzip

**Full script that generated the error**

```
tar -xzf SanFranSenDT42.tar.gz

```

**Full error message**

```
gzip: stdin: invalid compressed data--format violated
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now

```

**System information**

* Operating system: 
* Python environment:

**Comment by abcdedfde** [comment]
I'm learning the MintPY by EarthScope20203 tutorail, where I'm trapping in this problem. I would greatly appreciate it if you can solve my problem. Thanks.

**Comment by yunjunz** [comment]
It seems like the downloaded *.tar.fz file is corrupted.
